examples/dynamo_integration/.env.example

```
@@ -13,42 +13,85 @@
 # =============================================================================
-# Hugging Face Configuration
+# OPTIONAL VARIABLES - GPU Configuration
 # =============================================================================
-# Cache directory for HF models and datasets
-HF_HOME=~/.cache/huggingface
-# Your HuggingFace token to pull the dataset and model weights (account and necessary permissions required)
-HF_TOKEN=<your-huggingface-read-token>
+
+# GPU device IDs for unified mode (comma-separated)
+# Default: 0,1,2,3
+export DYNAMO_GPU_DEVICES="0,1,2,3"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)